src/Symfony/Bundle/TwigBundle/Command/DebugCommand.php

```diff
@@ -11,7 +11,9 @@
 
 namespace Symfony\Bundle\TwigBundle\Command;
 
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Bridge\Twig\Command\DebugCommand as BaseDebugCommand;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 use Symfony\Component\Console\Input\InputArgument;
```

Comment:
`InputArgument`, `InputInterface`, `InputOption` and `OutputInterface` are not used.